docs/src/main/mdoc/datatypes/writer.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: docs
+title:  "Writer"
+section: "data"
+source: "core/src/main/scala/cats/data/package.scala"
+scaladoc: "#cats.data.Writer"
+---
+# Writer
+
+The `Writer[L, A]` datatype represents a computation that produce a
+tuple containing a value of type `L` and one of type `A`. Usually, the
+value `L` represents a description of the computation. A typical
+example of an `L` value could be a logging `String` and that's why
+from now on we will refer to it as the _Logging side_ of the
+datatype. Meanwhile, the value `A` is the actual output of the
+computation.
+
+The main features the `Writer` provides are:
+- The flexibility regarding Log value management. It can be modified
+in multiple ways. See the [Operations section](#operations)
+- When two functions are composed together, eg using `flatMap`, the
+  logs of both functions will be combined using an implicit [Semigroup](https://typelevel.org/cats/typeclasses/semigroup.html).
```

Comment:
Both are good points, but if it's an enumeration, the structure of the sentences should be similar. I suggest removing the bullet points and rephrasing it as a paragraph.